dask/array/routines.py

```diff
@@ -998,11 +998,7 @@ def histogram(a, bins=None, range=None, normed=False, weights=None, density=None
     token = tokenize(a, bins, range, weights, density)
```

Comment:
While the approach is valid, there's a library function that might simplify this.